dev_guide/projects.adoc

```diff
@@ -165,3 +165,36 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
```

Comment:
Consider that the Dedicated and Online versions of this can't link to these install_config and admin_guide books.